app/javascript/src/components/common-prop-types/organization-prop-type.js

```diff
@@ -0,0 +1,6 @@
+import PropTypes from 'prop-types';
+
+export const organizationPropType = PropTypes.shape({
```

Comment:
Made an organization prop type that can be used throughout the project.

Reply:
Nice